src/scripts.js

```diff
@@ -85,21 +53,22 @@ function getUserById(id, listRepo) {
 
 
 function addInfoToSidebar(user, userStorage) {
-  sidebarName.innerText = user.name;
-  headerText.innerText = `${user.getFirstName()}'s Activity Tracker`;
-  stepGoalCard.innerText = `Your daily step goal is ${user.dailyStepGoal}.`
+  $('#sidebarName').text(user.name);
+  $('#headerText').text(`${user.getFirstName()}'s Activity Tracker`);
+  $('#stepGoalCard').text(`Your daily step goal is ${user.dailyStepGoal}.`);
+  // avStepGoalCard is never selected and used
```

Comment:
If it's never selected, then go ahead and remove the next line (and this comment, too)